app/src/main/res/layout/fragment_about.xml

```diff
@@ -58,7 +58,8 @@
                     android:layout_marginLeft="@dimen/spacing_negative"
                     android:layout_marginRight="@dimen/spacing_negative"
                     android:layout_marginStart="@dimen/spacing_negative"
-                    android:layout_marginTop="@dimen/spacing" />
+                    android:layout_marginTop="@dimen/spacing"
+                    android:layout_marginBottom="@dimen/spacing_small"/>
 
                 <TextView
                     style="@style/AboutTitle"
```

Comment:
Oh you are right, `marginTop` is frequently used.. I think using `marginTop` is more consistent as well! I'll fix it, thank you! :+1: